### DISCUSSION TITLE
Contributors: allow the option to see full list with names

### DISCUSSION THREAD

**Discussion by petya-kangalova** [hotosm/tasking-manager#6204] [General]
## Problem:
Aat the moment we cannot see the full list of contributors.  We only see the icons
![image](https://github.com/hotosm/tasking-manager/assets/98902727/37b22b1b-306f-403f-843b-bc399c742877)

## Proposed solutions:
We want to allow at least team manager to easily see the full list without having to click on each item. Suggestion will be to allow a CSV download with the contributors and they account page.
1. Main project page - Contributors
2. When you go into Team- Contributors - we would like see full list with account  page.  Suggestion to keep icons AND add "Download full contributor list" option in CSV.

@ramyaragupathy  for you to consider prioritisation.

**Comment by verdy-p** [comment]
Alternative, allow a rectangular sélection and multiselection of
nodes/icons on the mam press a buttons to get a tableview from which we can
export the selection in a basic csv or json file. The map could be like the
tool "Who dis this ?" (with its existing filters) . The table view, where
we could add/remove infos or with other filters, and then export the data
from the table view could be integrated there. The table view may need to
be paginated every ~200 rows.
